### PR TITLE
BXMSPROD-33 Create zip assembly with wars and config for Wildfly server

### PR DESCRIPTION
**Pull request by Ginxo**

### CODE REVIEW

File: business-central-parent/business-central-distribution-wars/business-central/pom.xml [hunk 1]
```diff
@@ -75,6 +75,32 @@
 
       <build>
         <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-dependency-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>get-business-central</id>
```
Comment (mareknovotny) [inline business-central-parent/business-central-distribution-wars/business-central/pom.xml]: I guess the id is incorrect ;), it gets wildfly distro not the BC

File: business-central-parent/business-central-distribution-wars/business-monitoring/pom.xml [hunk 2]
```diff
@@ -44,6 +44,32 @@
 
       <build>
         <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-dependency-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>get-business-central</id>
```
Comment (mareknovotny) [inline business-central-parent/business-central-distribution-wars/business-monitoring/pom.xml]: I guess the id is incorrect ;), it gets wildfly distro not the BC

**Comment by kie-ci2** [comment]
Build finished. 119 tests run, 5 skipped, 0 failed.

**Comment by mareknovotny** [comment]
@Ginxo one thing which I probably didn't emphasized is we should remove 'community/assembly-kie-wb-eap-7.xml'  to productized profile and there rename 'productized/assembly-kie-wb-eap-7-redhat.xml'
As EAP7 assembly makes sense only for productized profile

File: business-central-parent/business-central-distribution-wars/business-central/pom.xml [hunk 3]
```diff
@@ -87,6 +113,7 @@
                 <configuration>
                   <descriptors>
                     <descriptor>${project.parent.basedir}/src/main/community/assembly-kie-wb-wildfly-14.xml</descriptor>
+                    <descriptor>${project.parent.basedir}/src/main/community/assembly-kie-wb-wildfly-14.zip.xml</descriptor>
                     <descriptor>${project.parent.basedir}/src/main/community/assembly-kie-wb-eap-7.xml</descriptor>
```
Comment (mareknovotny) [inline business-central-parent/business-central-distribution-wars/business-central/pom.xml]: so this should go into productized profile i think

File: business-central-parent/business-central-distribution-wars/business-monitoring/pom.xml [hunk 4]
```diff
@@ -56,6 +82,7 @@
                 <configuration>
                   <descriptors>
                     <descriptor>${project.parent.basedir}/src/main/community/assembly-kie-wb-wildfly-14.xml</descriptor>
+                    <descriptor>${project.parent.basedir}/src/main/community/assembly-kie-wb-wildfly-14.zip.xml</descriptor>
                     <descriptor>${project.parent.basedir}/src/main/community/assembly-kie-wb-eap-7.xml</descriptor>
```
Comment (mareknovotny) [inline business-central-parent/business-central-distribution-wars/business-monitoring/pom.xml]: so this should go into productized profile i think

**Pull request closed without merging**